warning: in the working copy of 'targets/f7/api_symbols.csv', CRLF will be replaced by LF the next time Git touches it
diff --git a/targets/f7/api_symbols.csv b/targets/f7/api_symbols.csv
index 2bb63b88..da39dccb 100644
--- a/targets/f7/api_symbols.csv
+++ b/targets/f7/api_symbols.csv
@@ -1,5 +1,5 @@
 entry,status,name,type,params
-Version,+,87.1,,
+Version,+,88.0,,
 Header,+,applications/drivers/subghz/cc1101_ext/cc1101_ext_interconnect.h,,
 Header,+,applications/services/bt/bt_service/bt.h,,
 Header,+,applications/services/bt/bt_service/bt_keys_storage.h,,
@@ -314,6 +314,7 @@ Header,+,targets/furi_hal_include/furi_hal_sd.h,,
 Header,+,targets/furi_hal_include/furi_hal_speaker.h,,
 Header,+,targets/furi_hal_include/furi_hal_spi.h,,
 Header,+,targets/furi_hal_include/furi_hal_usb.h,,
+Header,+,targets/furi_hal_include/furi_hal_usb_airbridge.h,,
 Header,+,targets/furi_hal_include/furi_hal_usb_ccid.h,,
 Header,+,targets/furi_hal_include/furi_hal_usb_hid.h,,
 Header,+,targets/furi_hal_include/furi_hal_usb_hid_u2f.h,,
@@ -800,6 +801,8 @@ Function,+,bt_keys_storage_set_storage_path,void,"Bt*, const char*"
 Function,+,bt_keys_storage_update,_Bool,"BtKeysStorage*, uint8_t*, uint32_t"
 Function,+,bt_profile_restore_default,_Bool,Bt*
 Function,+,bt_profile_start,FuriHalBleProfileBase*,"Bt*, const FuriHalBleProfileTemplate*, FuriHalBleProfileParams"
+Function,+,bt_serial_tx,_Bool,"const uint8_t*, uint16_t"
+Function,+,bt_set_raw_serial_callback,void,"BtRawSerialCallback, void*"
 Function,+,bt_set_status_changed_callback,void,"Bt*, BtStatusChangedCallback, void*"
 Function,+,buffered_file_stream_alloc,Stream*,Storage*
 Function,+,buffered_file_stream_close,_Bool,Stream*
@@ -1418,6 +1421,9 @@ Function,+,furi_hal_gpio_init,void,"const GpioPin*, const GpioMode, const GpioPu
 Function,+,furi_hal_gpio_init_ex,void,"const GpioPin*, const GpioMode, const GpioPull, const GpioSpeed, const GpioAltFn"
 Function,+,furi_hal_gpio_init_simple,void,"const GpioPin*, const GpioMode"
 Function,+,furi_hal_gpio_remove_int_callback,void,const GpioPin*
+Function,+,furi_hal_hid_airbridge_kb_press,_Bool,uint16_t
+Function,+,furi_hal_hid_airbridge_kb_release,_Bool,uint16_t
+Function,+,furi_hal_hid_airbridge_kb_release_all,_Bool,
 Function,+,furi_hal_hid_consumer_key_press,_Bool,uint16_t
 Function,+,furi_hal_hid_consumer_key_release,_Bool,uint16_t
 Function,+,furi_hal_hid_consumer_key_release_all,_Bool,
@@ -1435,6 +1441,11 @@ Function,+,furi_hal_hid_u2f_get_request,uint32_t,uint8_t*
 Function,+,furi_hal_hid_u2f_is_connected,_Bool,
 Function,+,furi_hal_hid_u2f_send_response,void,"uint8_t*, uint8_t"
 Function,+,furi_hal_hid_u2f_set_callback,void,"HidU2fCallback, void*"
+Function,+,furi_hal_hid_vendor_get_request,uint32_t,uint8_t*
+Function,+,furi_hal_hid_vendor_is_connected,_Bool,
+Function,+,furi_hal_hid_vendor_send_response,_Bool,"uint8_t*, uint8_t"
+Function,+,furi_hal_hid_vendor_send_response_blocking,_Bool,"uint8_t*, uint8_t, uint32_t"
+Function,+,furi_hal_hid_vendor_set_callback,void,"HidVendorCallback, void*"
 Function,+,furi_hal_i2c_acquire,void,const FuriHalI2cBusHandle*
 Function,-,furi_hal_i2c_deinit_early,void,
 Function,-,furi_hal_i2c_init,void,
@@ -1741,6 +1752,13 @@ Function,+,furi_hal_subghz_stop_async_tx,void,
 Function,+,furi_hal_subghz_tx,_Bool,
 Function,+,furi_hal_subghz_write_packet,void,"const uint8_t*, uint8_t"
 Function,+,furi_hal_switch,void,void*
+Function,+,furi_hal_usb_airbridge_get_profile,FuriHalUsbInterface*,uint8_t
+Function,+,furi_hal_usb_airbridge_profile_count,uint8_t,
+Function,+,furi_hal_usb_airbridge_profile_has_keyboard,_Bool,uint8_t
+Function,+,furi_hal_usb_airbridge_profile_identity,const char*,uint8_t
+Function,+,furi_hal_usb_airbridge_profile_label,const char*,uint8_t
+Function,+,furi_hal_usb_airbridge_profile_pid,uint16_t,uint8_t
+Function,+,furi_hal_usb_airbridge_profile_vid,uint16_t,uint8_t
 Function,+,furi_hal_usb_ccid_insert_smartcard,void,
 Function,+,furi_hal_usb_ccid_remove_smartcard,void,
 Function,+,furi_hal_usb_ccid_set_callbacks,void,"CcidCallbacks*, void*"
@@ -4155,6 +4173,7 @@ Variable,+,subghz_protocol_raw_decoder,const SubGhzProtocolDecoder,
 Variable,+,subghz_protocol_raw_encoder,const SubGhzProtocolEncoder,
 Variable,+,subghz_protocol_registry,const SubGhzProtocolRegistry,
 Variable,-,suboptarg,char*,
+Variable,+,usb_airbridge,FuriHalUsbInterface,
 Variable,+,usb_ccid,FuriHalUsbInterface,
 Variable,+,usb_cdc_dual,FuriHalUsbInterface,
 Variable,+,usb_cdc_single,FuriHalUsbInterface,
